animations/test/misc/expand_card_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/misc/expand_card.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createExpandCardScreen() => MaterialApp(
+      home: ExpandCardDemo(),
+    );
+
+void main() {
+  group('ExpandCard Tests', () {
+    testWidgets('ExpandCard changes size on tap', (tester) async {
+      await tester.pumpWidget(createExpandCardScreen());
+
+      // Get the initial size of ExpandCard.
+      Size initialSize = tester.getSize(find.byType(ExpandCard));
+
+      // Tap on the ExpandCard.
+      await tester.tap(find.byType(ExpandCard));
+      await tester.pumpAndSettle();
+
+      // The size of ExpandCard must change once tapped.
+      expect(
+        initialSize,
+        isNot(equals(tester.getSize(find.byType(ExpandCard)))),
```

Comment:
There's a matcher called `greaterThan` that would probably be a better fit here than `isNot(equals(...))`

Reply:
Thanks :)
I have updated the test with `lessThan` matcher and added a comment.